config/sync/shortcut.set.default.yml

```diff
@@ -1,4 +1,4 @@
-uuid: NULL
+uuid: 51ee828c-3016-49a8-84cb-9c778cb62cbc
```

Comment:
@brandtziegler use git add -p to make sure you don't add extra, unrelated work to the pull request. This portion of the commit re-introduces the core shortcut bug